src/components/ShoppingCart.vue

```diff
@@ -47,11 +47,7 @@
 				return this.$store.getters.cartItemList;
 			},
 			totalValue() {
-				let res = 0;
-				this.cartItemList.map((item, idx) => {
-					res += item.price * item.quantity;
-				});
-				return res;
+				return this.$store.getter.cartValue;
```

Comment:
I think it should be `return this.$store.getters.cartValue`.
Can you use `mapGetters` instead?